README.md

```diff
@@ -110,9 +110,9 @@ const ObjectId = Schema.ObjectId;
 
 const BlogPost = new Schema({
   author: ObjectId,
-  title: String,
-  body: String,
-  date: Date
+  title:  String,
+  body:   String,
```

Comment:
Can we change this particular setting? This extra indentation is inconsistent with how our docs currently define schemas, and I don't think it is worth switching.

Reply:
i had initially added the setting, because i had found some documentation where it was indented to be on the same "column", but also thought that it would be good to keep one rule instead of disabling / enabling for specific sections

but yes, i can remove that rule again and format all documentation with only one space instead of being aligned

Reply:
rule removed & applied in 88eec11d848363cc0610610dfec9c826c84eca13

Reply:
i had it originally added, because it was in use at `schematypes.md`, see changes with the rule removed: https://github.com/Automattic/mongoose/pull/13156/files#diff-2f4335ef76e27dfe5e52253d5a42778f66bdf448427004d0b0f4ae927a2b424f